### PR TITLE
adds image-to-gif example #98

### PR DESCRIPTION
**Pull request by ablwr**
Many images into a gif, as requested!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -339,6 +339,32 @@
...
+          <h3>Images to GIF</h3>
+          <p><code>ffmpeg -f image2 -framerate 9 -pattern_type glob -i <i>"input_image_*.jpg"</i> -vf scale=250x250 <i>output_file</i>.gif</code></p>
+          <p>This will convert a series of image files into a gif.</p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-f image2</dt><dd>forces input or output file format. <code>image2</code> specifies the image file demuxer.</dd>
+            <dt>-framerate 9</dt><dd>sets framerate to 9 frames per second</dd>
+            <dt>-pattern_type glob</dt><dd>tells ffmpeg that the following mapping should "interpret like a <a href="https://en.wikipedia.org/wiki/Glob_%28programming%29">glob</a>"</dd>
+            <dt>-i <i>"input_image_*.jpg"</i></dt><dd>maps all files in the directory that start with input_image_, for example input_image_001.jpg, input_image_002.jpg, input_image_003.jpg... etc.<br/>
+            (The quotation marks are necessary!)</dd>
```
Comment (retokromer) [inline index.html]: For didactical reasons, I would add why they are necessary (as a glob, for a glob, because it’s a glob _aut similia)_.

**Review by retokromer — APPROVED** [review]
See comment. LGTM

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html]: Thanks, Reto! I agree although had a little difficultly trying to succinctly summarize "glob." Updated, let me know what ya think.

File: index.html [hunk 2]
```diff
@@ -352,9 +352,9 @@
             <dt>ffmpeg</dt><dd>starts the command</dd>
             <dt>-f image2</dt><dd>forces input or output file format. <code>image2</code> specifies the image file demuxer.</dd>
             <dt>-framerate 9</dt><dd>sets framerate to 9 frames per second</dd>
-            <dt>-pattern_type glob</dt><dd>tells ffmpeg that the following mapping should "interpret like a <a href="https://en.wikipedia.org/wiki/Glob_%28programming%29">glob</a>"</dd>
+            <dt>-pattern_type glob</dt><dd>tells ffmpeg that the following mapping should "interpret like a <a href="https://en.wikipedia.org/wiki/Glob_%28programming%29">glob</a>" (a "global command" function that relies on the * as a wildcard and finds everything that matches</dd>
```
Comment (retokromer) [inline index.html]: )

**Review by retokromer — CHANGES_REQUESTED** [review]

**Comment by retokromer** [comment]
This is useful, thank you!

**Comment by ablwr** [comment]
Thank @kgrons !

**Pull request merged**